pkg/systemd/logs.js

```diff
@@ -21,13 +21,23 @@ import $ from "jquery";
 import cockpit from "cockpit";
 import { journal } from "journal";
 
+const PromptType = Object.freeze({
+    ASK: 0,
+    ASK_YES_NO: 1,
+    ASK_YES_NO_YESFOREVER: 2,
+    ASK_YES_NO_SAVE: 3,
+    ASK_PASSWORD: 4,
+});
+
 $(function() {
     cockpit.translate();
     const _ = cockpit.gettext;
 
     var problems_client = cockpit.dbus('org.freedesktop.problems', { superuser: "try" });
+    var reportd_client = cockpit.dbus('org.freedesktop.reportd', { bus: 'session' });
```

Comment:
What about root user that doesn't have session bus?

Reply:
Better put in right before the reportd variable, instead of mixing with the problems one.

Reply:
That is something I missed completely. reportd will only listen on the session bus, so that’s a bit of a problem. I can get it to work by using a bus-activatable systemd service unit, but it’s far from a good user experience, as the proxy creation fails initially, but after a refresh it’s fine.

Something to figure out still.